lib/cloudant.js

```diff
@@ -6,7 +6,7 @@
 'use strict';
 
 var g = require('strong-globalize')();
-var Connector = require('loopback-connector').Connector;
+var CouchDB = require('loopback-connector-couchdb2').CouchDB;
```

Comment:
Do we? I thought the only functions needing them is all in Couch so it will use the one in there.